src/crypto-browser.js

```diff
@@ -34,9 +35,10 @@ function webCryptoHash (type) {
       return
     }
 
-    return res.then((arrbuf) => {
-      callback(null, new Buffer(new Uint8Array(arrbuf)))
-    }).catch((err) => callback(err))
+    nodeify(
+      res.then((raw) => new Buffer(new Uint8Array(raw))),
+      callback
+    )
```

Comment:
no, we create a new promise using the then which is passed as the first arg to nodeify and the callback is passed as the second argument, as described in the nodeify docs